src/dyad/core/dyad_core.c

```diff
@@ -516,7 +523,8 @@ DYAD_CORE_FUNC_MODS dyad_rc_t dyad_cons_store (const dyad_ctx_t* restrict ctx,
     }
     rc = DYAD_RC_OK;
 
-pull_done:
+pull_done:;
```

Comment:
We restructured consumer core code to use file locking. file_data is deallocated at consume_done rather than pull_done